packages/lib-sourcify/src/Validation/SolidityMetadataContract.ts

```diff
@@ -236,11 +242,12 @@ export class SolidityMetadataContract {
       Object.keys(this.missingSources).length > 0 ||
       Object.keys(this.invalidSources).length > 0
     ) {
-      throw new Error(
-        `Can't create JsonInput from metadata: Missing or invalid sources in metadata: ${JSON.stringify(
+      logDebug(
+        `Can't create JsonInput from metadata: Missing or invalid sources in metadata: missing:${JSON.stringify(
           this.missingSources,
-        )}`,
+        )}; invalid: ${JSON.stringify(this.invalidSources)}`,
       );
+      throw new ValidationError('missing_or_invalid_source');
```

Comment:
Are we passing the missing sources to the user? Idk what we used to pass before but it feels like it's an important info for the user (missing keccak / missing path)

Reply:
Same issue here. We don't store the error message in the db.